json_schema.py

```diff
@@ -0,0 +1,58 @@
+from dtos import CriteriaDto
+
+
+class JsonSchemaLoader:
```

Comment:
should be a part of an array of a set of criteria. maybe add smth like schema builder: given a set of criteria and a target property, we should add this single criteria schema to the final output schema being built